stable/object-bucket-claim/templates/object-bucket-claim.yaml

```diff
@@ -103,12 +103,25 @@ spec:
               esac
 
               # check the consumer app HelmRelease exists before kicking off the reconcile
-              kubectl get helmrelease -n kommander {{ $app }}
+              kubectl get helmrelease -n {{ $.Release.Namespace }} {{ $app }}
               if [[ $? -eq 1 ]]; then
                   echo "HelmRelease {{ $app }} doesn't exist. Do nothing."
                   exit 0
               fi
 
+              # detect helm storage deadlock
+              # https://github.com/fluxcd/helm-controller/issues/149
+              helm_deadlock_msg="Helm upgrade failed: another operation (install/upgrade/rollback) is in progress"
+              kubectl get helmrelease -n {{ $.Release.Namespace }} {{ $app }} -o json | \
+              jq -e --arg helm_deadlock_msg $helm_deadlock_msg '.status | .conditions | map(.message) | contains([$helm_deadlock_msg])'
```

Comment:
Env var `$helm_deadlock_msg` will not expand in single quotes, also might need to double quote it in here.

Reply:
yeah this looks a bit confusing but `$helm_deadlock_msg` in the `jq` query comes from `--arg helm_deadlock_msg $helm_deadlock_msg`, otherwise jq will complain. I did some local test with a json file and it works.

Reply:
Oh sorry I misread that

Reply:
well actually you are right on the quote, its needed in `--arg helm_deadlock_msg "$helm_deadlock_msg"`